packages/medulas-react-components/src/context/ToastProvider/Toast/index.tsx

```diff
@@ -29,14 +29,13 @@ export const Toast = ({
 }: Props): JSX.Element => {
   return (
     <Snackbar
+      id="toast-provider"
       anchorOrigin={anchorProps}
       open={open}
       autoHideDuration={5000}
       onClose={onClose}
     >
-      {open && (
-        <ToastContent onClose={onClose} variant={variant} message={message} />
-      )}
```

Comment:
This code throw error in case if `open===false` because of empty child.

Reply:
I know, I was aware, the thing is, removing the boolean condition, you see a strange effect in the component for a second while it is being closed (you can reproduce it in storybook), so some extra stuff needs to be done inside ToastContent.

Reply:
Can you take a look into it?

Reply:
Done